weblate-scripts/pre_commit.py

```diff
@@ -43,13 +43,14 @@ def properties_to_xwiki_xml(file_path, path_prefix, lang):
     with open(properties_path, "r") as f_properties:
         properties.load(f_properties.read())
 
-    title = properties.get_value("{}.title".format(file_name))
-    content = properties.get_value("{}.content".format(file_name))
-    xml_file = XmlFile()
-    xml_file.load(path_prefix + file_path)
-    xml_file.set_tag_content("title", title)
-    xml_file.set_tag_content("content", content)
-    xml_file.write(path_prefix + file_path)
+    if not properties.is_empty():
```

Comment:
Instead you can explicitly `return` the method and add a log message (a simple print, or something else if you have time).

Reply:
ok for adding a log in a else but I don't really understand what you mean by 
>  you can explicitly return the method

Reply:
Hm. I prefer handling the not case here, I find it clearer.